samples/test/support/check/checkURLPath.ts

```diff
@@ -10,7 +10,7 @@ export default async (falseCase: boolean, expectedPath: string) => {
      * @type {String}
      */
     let currentUrl = await browser.getUrl();
-    currentUrl = currentUrl.replace(/http(s?):\/\//, '');
+    currentUrl = currentUrl.replace(/http(s?):\/\//, '').replace(/#(.*?)$/, '');
```

Comment:
Facebook redirects to `/#_=_`

Reply:
Can you explain more about what happens here? My understanding is the app redirect to facebook for SSO, then facebook redirect to okta, then okta redirects to the app.

Reply:
Yes, your understanding is correct.
Facebook redirects to `https://javascript-idx-sdk.okta.com/oauth2/v1/authorize/callback?code=AAA#_=_`.
Which redirects to `http://localhost:8080/login/callback?interaction_code=AAA&state=AAA`
Then I see home page with URL `http://localhost:8080/#_=_`.

Reply:
I see, one idea is to make checkURLPath accept a `transformUrl` function, then we can customize the expected url based on different scenarios, so some specially handling won't affect the general cases.

Reply:
I've added 3rd param `removeHash` (default false) to make it simpler, if you don't mind